### PR TITLE
Add Prime_E to Via

### PR DESCRIPTION
**Pull request by holtenc**
@Wilba6582 and I discussed adding this last night. 

I apologize for the messy commit history. I'm not very familiar with github and the associated best-use practices.

### CODE REVIEW

**Comment by holtenc** [comment]
corresponding QMK PR: https://github.com/qmk/qmk_firmware/pull/5285

**Comment by drashna** [comment]
QMK Firmware pull request merged.

File: app/utils/device-meta.js [hunk 1]
```diff
@@ -238,7 +240,13 @@ export const DEVICE_META_MAP: DeviceMetaMap = {
...
     matrixLayout: MATRIX_G60,
     lights: false,
-  }
+  },
+  [0x50520051]: {
+    name: 'PRIME_E',
+    layout: LAYOUT_PRIME_E,
+    matrixLayout: MATRIX_PRIME_E,
+    lights: false
+    }
```
Comment (olivia) [inline app/utils/device-meta.js]: Fix spacing of `}` please

Comment (holtenc) [reply app/utils/device-meta.js]: done.

**Comment by holtenc** [comment]
updated fork after changes merged from https://github.com/olivia/via-config/pull/53. This should resolve conflicts in device-meta.js

**Comment by olivia** [comment]
It appears that there are still conflicts?

**Comment by holtenc** [comment]
If there are still conflicts github is failing to show them on my side. 

capture.

could you be more specific?

**Comment by drashna** [comment]
@olivia I know that GitHub will report conflicts based on the merge type (merge vs squash vs rebase).  It's ... annoying, IMO. And that may be the issue

**Comment by holtenc** [comment]
at this point I'm not really what to do about the conflicts you're getting as I can't see them nor do I have any details about what exactly is conflicting. Only Idea I have is to close this PR and I can start with a fresh VIA code base, add my changes and re-submit the PR. Unless you guys have a better idea. I'm all ears.

**Comment by holtenc** [comment]
closing this to resubmit from a fresh clone and install of via-config in an attempt to resolve whatever the conflicts are.

**Pull request closed without merging**